src/v2/bigtable_instance_admin_client.js

```diff
@@ -127,6 +127,11 @@ class BigtableInstanceAdminClient {
         'nextPageToken',
         'appProfiles'
       ),
+      listClusters: new gax.PageDescriptor(
```

Comment:
@alexander-fenster  I had to manually add this to enable the auto-pagination behavior that other `list` endpoints have.

Reply:
this is the second time we edit something in the generated code (the first case was in pubsub here https://github.com/googleapis/nodejs-pubsub/blob/d0dc20ceddf6a897d5143d5342ea1d084d0da57d/src/v1/subscriber_client.js#L225) and we might face problems if we continue doing it. At least we need to make sure that unit tests will fail if this is not defined here (in case if someone regenerates and overwrites the code).

Reply:
I definitely didn't want to leave this here. Can we instead fix the generator?

Reply:
Took a closer look - it must be fixed in [here](https://github.com/googleapis/googleapis/blob/d90956b8adf6c820969457d848db833328076879/google/bigtable/admin/v2/bigtableadmin_gapic.yaml#L200), not in the generator. Working on it.

Reply:
@stephenplusplus, I just got an explanation from bigtable owners (@igorbernstein2). Neither listClusters nor listInstances should be paginaged, and the page tokens for these methods will be removed in v3. The reasoning is: (a) pagination of clusters is token is complicated on the server side, and (b) these list will never grow too large. So the pagination code is not generated here for a reason.

Reply:
Alright, fixing now.